locale/fr/libbout.po

```diff
@@ -17,94 +17,684 @@ msgstr ""
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n > 1);\n"
 
-#: src/bout++.cxx:365
+#: ../src/mesh/impls/bout/boutmesh.cxx:347
```

Comment:
Thanks for updating the translations. Sorry I think the entries where `msgstr` is empty should be deleted. If they're deleted then I think gettext keeps the original.

Reply:
Ah sorry, my mistake. Some confusion in online fora, but the consensus seems to be that empty strings are counted as untranslated and so ignored.